addons/mail/static/src/chatter/web_portal/chatter.js

```diff
@@ -9,8 +9,8 @@ import { Activity } from "@mail/core/web/activity";
 import { SuggestedRecipientsList } from "@mail/core/web/suggested_recipient_list";
 import { useHover, useMessageHighlight } from "@mail/utils/common/hooks";
 import { isDragSourceExternalFile } from "@mail/utils/common/misc";
-import { RecipientList } from "./recipient_list";
-import { FollowerList } from "./follower_list";
+import { RecipientList } from "@mail/core/web/recipient_list";
+import { FollowerList } from "@mail/core/web/follower_list";
```

Comment:
It looks like these 2 should be moved to chatter subfolder as well (not used anywhere else?)